tests/foreman/cli/test_oscap_tailoringfiles.py

```diff
@@ -206,15 +207,17 @@ def test_positive_download_tailoring_file(self):
 
         :expectedresults: The tailoring file should be downloaded
 
+        BZ: 1857572
+
         :CaseImportance: Critical
         """
         name = gen_string('alphanumeric')
-        file_path = '/tmp/{0}.xml'.format(name)
+        file_path = f'/var/{self.tailoring_file_path}'
         tailoring_file = make_tailoringfile({'name': name, 'scap-file': self.tailoring_file_path})
         assert tailoring_file['name'] == name
-        result = TailoringFiles.download_tailoring_file({'name': name, 'path': '/tmp/'})
+        result = TailoringFiles.download_tailoring_file({'name': name, 'path': '/var/tmp/'})
         assert file_path in result[0]
-        result = ssh.command('find {0}'.format(file_path))
+        result = ssh.command(f'find {file_path}')
```

Comment:
In find command, It is always good to redirect the unwanted error into /dev/null, But I am fine with the current implementation.

Reply:
@devendra104 updated PR.